cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs

```diff
@@ -1027,7 +1055,8 @@ writeStakeAddressInfo
                     , "stakeDelegation" .= mPoolId
                     , "voteDelegation" .= fmap friendlyDRep mDRep
                     , "rewardAccountBalance" .= mBalance
-                    , "delegationDeposit" .= mDeposit
+                    , "stakeRegistrationDeposit" .= mDeposit
+                    , "govActionDeposits" .= gaDeposits
                     ]
```

Comment:
I'm wondering if it wouldn't be better to skip those fields altogether like:
```suggestion
                    ]
                    <> (monoidForEraInEon @ConwaEraOnwards (toCardanoEra beo) $ 
                          const
                           [ "stakeRegistrationDeposit" .= mDeposit
                           , "govActionDeposits" .= gaDeposits
                           ]
                       )
```

I'm not sure how should the result:
```json
... "stakeRegistrationDeposit": null, "govActionDeposits": [] }
```
be interpreted in eras < Conway.

Reply:
When the node is in babbage era the query does not return these fields. If the node is in Conway era, the query returns the fields even when using `babbage query`. Which, make sense. Also, since we are deprecating eras < Conway we do not need to make any changes here.

Reply:
Thanks for the explanation, makes sense I think. 👍🏻